optimize/backend/src/it/java/io/camunda/optimize/service/db/es/report/util/HyperMapAsserter.java.out

```diff
@@ -25,7 +25,6 @@
 // import java.util.Optional;
 // import lombok.RequiredArgsConstructor;
```

Comment:
For the moment I'm not touching the files that are commented out (in fact, I am going to restore the ones that I changed): if they will be deleted, then it's a no-problem, otherwise, they will need to be de-lombok'ed later, and for this, we need to keep their annotations and import, for the moment.